src/components/docs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,28 @@
+import { Breadcrumb, BreadcrumbItem, BreadcrumbLink } from '@chakra-ui/react';
+import { FC } from 'react';
+
+interface Props {
+  router: any
+}
+
+export const Breadcrumbs: FC<Props> = ({ router }) => {
+  let pathSplit = router.asPath.split('/')
+  pathSplit = pathSplit.splice(1,pathSplit.length)
+
+  return (
+    <Breadcrumb mb={10}>
+      {pathSplit.map((path: string, idx: number) => {
+        return (
+          <BreadcrumbItem key={path}>
+            <BreadcrumbLink
```

Comment:
should be wrapped inside a `next/link` component to avoid full page reload

Reply:
Sounds good, will try out.